windows-agent/internal/proservices/wslinstance/landscapeconfig.go

```diff
@@ -0,0 +1,88 @@
+package wslinstance
+
+import (
+	"errors"
+	"fmt"
+
+	agentapi "github.com/canonical/ubuntu-pro-for-wsl/agentapi/go"
+	log "github.com/canonical/ubuntu-pro-for-wsl/common/grpc/logstreamer"
+	"github.com/ubuntu/decorate"
+)
+
+// LandscapeConfigCommands serves the homonymous stream.
+func (s *Service) LandscapeConfigCommands(stream agentapi.WSLInstance_LandscapeConfigCommandsServer) (err error) {
+	defer decorate.OnError(&err, "WslInstance: could not handle landscape config commands")
+	ctx := stream.Context()
+
+	client, err := commandHandshake(ctx, s, stream.Recv)
+	if err != nil {
+		return err
+	}
+	if err := client.SetLandscapeConfigStream(stream); err != nil {
+		return err
+	}
+	defer client.Close()
+
+	if err := client.WaitReady(ctx); err != nil {
+		return err
+	}
+
+	// Block until the connection drops
+	client.WaitDone(ctx)
+	return nil
+}
+
+// SetLandscapeConfigStream sets the landscape config stream for the client.
```

Comment:
```suggestion
// SetLandscapeConfigStream sets the Landscape config stream for the client.
```

I think we should capitalize all instances of the word `Landscape`.